bodhi/server/consumers/masher.py

```diff
@@ -406,23 +403,9 @@ def unlock_updates(self):
         self.log.debug('Unlocking updates')
         for update in self.updates:
             update.locked = False
+            update.date_locked = None
         self.db.flush()
 
-    def locked_date_for_update(self):
-        """ Return the date & time when an update has been locked """
-        for update in self.updates:
-            # Return datetime when an update is locked and datetime is
-            # not yet assigned to the column date_locked
-            if update.locked and not update.date_locked:
-                update.date_locked = datetime.utcnow()
-                self.db.commit()
-                self.db.refresh(update)
-                return update.date_locked
```

Comment:
This seems like a pretty strange pattern - to loop over updates until you hit one that's locked without a date, set its date, and… return? What about the rest of the updates in ```self.updates```?